Library/include/CSP/Systems/EventTicketing/EventTicketing.h

```diff
@@ -41,6 +48,23 @@ class CSP_API TicketedEvent
 	bool IsTicketingActive;
 };
 
+/// @ingroup Event Ticketing System
+/// @brief Data representation of a ticket for an event
+class CSP_API EventTicket
+{
+public:
+	EventTicket() : Vendor(EventTicketingVendor::Unknown), Status(TicketStatus::Unknown) {};
+
+	csp::common::String Id;
```

Comment:
Can we get inline doxygen formatted comments on these member vars explaining their purpose?